solitude/settings/base.py

```diff
@@ -510,3 +510,7 @@
 
 # An arbitrary limit on the number of payment methods.
 BRAINTREE_MAX_METHODS = 5
+
+# Arbitrary amounts for top and bottom limits.
+BRAINTREE_MAX_AMOUNT = 10000
+BRAINTREE_MIN_AMOUNT = 1
```

Comment:
A minimum definitely, otherwise Braintree will charge us more than we donate if its below about 30 cents. A maximum? Dunno.